src/controller/CHIPDeviceController.cpp

```diff
@@ -768,13 +768,13 @@ DeviceCommissioner::ContinueCommissioningAfterDeviceAttestationFailure(DevicePro
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR DeviceCommissioner::GetAttestationChallenge(ByteSpan & attestationChallenge)
+CHIP_ERROR DeviceCommissioner::GetAttestationChallenge(DeviceProxy * device, ByteSpan & attestationChallenge)
```

Comment:
Does this method now use `this` in any way? Should it be marked static in the header if it never uses this (and eventually moved around/refactored)?